src/types/Sort.ts

```diff
@@ -0,0 +1,7 @@
+export enum Sort {
+  name = 'name',
```

Comment:
Consider using uppercase for enum names to follow naming [conventions](https://michaelcurrin.github.io/dev-cheatsheets/cheatsheets/typescript/data-types/enums.html#:~:text=Naming%20convention,enum%20tend%20to%20be%20titlecase.).